Jenkinsfile

```diff
@@ -0,0 +1,15 @@
+node {
+	stage 'checkout'
+	git url: "https://github.com/azavea/raster-foundry.git", 
+	branch: "feature/jenkins-pipeline"
+
+	stage 'build'
+	sh "sleep 3;"
+	
+	stage 'test'
```

Comment:
Since the tests are executed within `cibuild`, it may make sense to rename this `stage` to `cibuild` with just the `cibuild` execution. Underneath, we can have a `cipublish` stage that just executes `cipublish`.